cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -17,16 +17,34 @@ limitations under the License.
 package cmd
 
 import (
+	"html/template"
 	"io"
 
 	"github.com/renstrom/dedent"
 	"github.com/spf13/cobra"
 
+	"strings"
+
+	"bytes"
+
 	"k8s.io/apiserver/pkg/util/flag"
+	"k8s.io/client-go/pkg/api"
+	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
+	kubeadmapiext "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1alpha1"
 	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases"
+	"k8s.io/kubernetes/cmd/kubeadm/app/images"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 )
 
+const (
+	// kubeadmUsageTemplateRequiredImages is appended to default usage template,
+	//so that user can see information about how to get required images.
```

Comment:
missing space added.
about scaling and adding more help subcommands, i will talk later.